rllib/algorithms/leela_chess_zero/__init__.py

```diff
@@ -0,0 +1,17 @@
+from ray.rllib.algorithms.leela_chess_zero.leela_chess_zero import (
+    LeelaChessZero,
+    LeelaChessZeroConfig,
+    LeelaChessZeroDefaultCallbacks,
+    DEFAULT_CONFIG,
```

Comment:
no need to import DEFAULT_CONFIG again. We have moved away from dict configs and are using AlgorithmConfigs now.